Sources/App/Migrations/071/UpdateVersionResetResolvedDependencies.swift

```diff
@@ -0,0 +1,34 @@
+// Copyright Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Fluent
+import SQLKit
+
+
+struct UpdateVersionResetResolvedDependencies: AsyncMigration {
+    func prepare(on database: Database) async throws {
+        // reset field values by dropping and re-creating the field
+        try await database.schema("versions")
+            .deleteField("resolved_dependencies")
+            .update()
+        try await database.schema("versions")
+            .field("resolved_dependencies",
+                   .array(of: .json))
+            .update()
```

Comment:
Same here, we really need to reset those values, because we can't know what `{}` in the field means.